.circleci/scripts/collect-har-artifact.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -x
+
+mkdir -p build-artifacts/yarn-install-har
+mv *.har build-artifacts/yarn-install-har/
```

Comment:
```suggestion
mv ./*.har build-artifacts/yarn-install-har/
```

ShellCheck is suggesting this change to prevent filenames from being interpreted as flags (https://github.com/koalaman/shellcheck/wiki/SC2035)